drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -87,13 +115,17 @@
             <section class="tab-content">
                 <form class="display-labels-form">
                     <h3>{% trans "Zone labels" %}</h3>
-                    <label for="display-labels">{% trans "Display label names on the image" %}:</label>
-                    <input name="display-labels" id="display-labels" type="checkbox" />
+                    <label>
+                        <span>{% trans "Display label names on the image" %}:</span>
```

Comment:
@cahrens I think having the checkbox before the text is more natural, so I swapped them and removed the colon.